.github/workflows/static_checks.yml

```diff
@@ -205,6 +207,16 @@ jobs:
         run: |
           bazel run //scripts:string_resource_validation_check -- $(pwd)
 
+      - name: Binary files check
+        run: |
+          bash /home/runner/work/oppia-android/oppia-android/scripts/pre-commit.sh
+          exit_code=$?
+
+          if [ $exit_code -eq 0 ]; then
```

Comment:
Should we fail this if the check fails? It seems like it always passes unless I'm missing something.

If the script failure already triggers the failure of the workflow, perhaps we can simplify this:

```sh
bash /home/runner/work/oppia-android/oppia-android/scripts/pre-commit.sh

# If the script above doesn't fail, then there are no binary files.
echo "No binary files found in commit."
```

Reply:
Thanks! Yes, the script is causing the failure. I've removed the redundant check for the pass statement.